generators/chaincode/templates/javascript/lib/contract-a.js

```diff
@@ -15,20 +15,31 @@
  * limitations under the License.
  */
 
-import { ChaincodeInterface, ChaincodeStub, Shim } from 'fabric-shim';
+'use strict';
 
-export class Chaincode implements ChaincodeInterface {
+const { Contract } = require('fabric-contract-api');
 
-    public async Init(stub: ChaincodeStub): Promise<any> {
-        const { fcn, params } = stub.getFunctionAndParameters();
-        console.info('Init()', fcn, params);
-        return Shim.success();
+const assetType = 'WIDGETS';
+
+class MyContract extends Contract {
+
+    constructor(){
+        super('org.example.mycontract');
     }
 
-    public async Invoke(stub: ChaincodeStub): Promise<any> {
-        const { fcn, params } = stub.getFunctionAndParameters();
-        console.info('Invoke()', fcn, params);
-        return Shim.success();
+    async transactionA(ctx,assetid,value) {
```

Comment:
We should drop the "asset" stuff and state interactions - just have some generic arguments that get logged to the console (e.g. arg1, arg2, arg3).